testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1120,6 +1120,75 @@ QUnit.test("searchEditorOptions", function(assert) {
     assert.strictEqual(searchEditorInstance.option("placeholder"), "Test", "placeholder of the search box");
 });
 
+QUnit.test("apply list search options if dataSource set as dataSource instance", function(assert) {
```

Comment:
It is not necessary to apply dataSource options to list. Widget should ignore dataSource options and take into attention list options only as it is done in DropDownList.